src/coreclr/src/tools/Common/TypeSystem/Common/ExceptionStringID.cs

```diff
@@ -16,6 +16,7 @@ public enum ExceptionStringID
         ClassLoadExplicitLayout,
         ClassLoadValueClassTooLarge,
         ClassLoadRankTooLarge,
+        ClassLoadUninstantiatedType,
```

Comment:
In think we can get by with ClassLoadGeneral. CoreCLR also doesn't have a special resource string for this case.